projects/gateway-utilities/ignition/script-python/General/Perspective/Table/code.py

```diff
@@ -0,0 +1,94 @@
+"""
+General.Perspective.Table
+
+This script contains functions used with table components.
+"""
+
+def export_table_to_csv(table):
```

Comment:
Can we actually change this to `to_xlsx` and then throw in a copy of it for CSV as well?